.github/workflows/ci.yml

```
@@ -10,28 +10,30 @@ jobs:
   main:
     runs-on: ubuntu-latest
     env:
+      CGO_ENABLED: 0
       GO_VERSION: '1.20'
       GOLANGCI_LINT_VERSION: v1.62.0
```

Let's not change the original order:
```suggestion
      GO_VERSION: '1.20'
      GOLANGCI_LINT_VERSION: v1.62.0
      CGO_ENABLED: 0
```